.travis.yml

```diff
@@ -20,6 +20,7 @@ cache:
 branches:
   only:
     - master
+    - rnmobile/master
```

Comment:
Yes, let's do it if you plan to use `rnmobile/master` more often 👍

Reply:
We might want to revert that I'm not sure. I merely merged `rnmobile/master` in here (thus the amount of changes unrelated to the PR description) but it might need some work before we merge it back to master

Reply:
We might actually need to branch off again as we focus on the Open Beta. Keeping it for now 👍

Reply:
I'm sure you will branch more often so it's fine to keep it here until we merge the mobile repo in here.